nload/plan.sh

```diff
@@ -0,0 +1,16 @@
+pkg_name=nload
+pkg_description="Real time network traffic monitor for the text console"
+pkg_origin=core
+pkg_version="0.7.4"
+pkg_maintainer="The Habitat Maintainers <humans@habitat.sh>"
+pkg_license=("GPL-2.0")
+pkg_source="https://github.com/rolandriegel/${pkg_name}/archive/v${pkg_version}.tar.gz"
+pkg_shasum="a73b3a75356776860fc4c40daebce04c5022f73d39704a12fb0aeb88a751216a"
+pkg_upstream_url="https://linux.die.net/man/1/nload"
```

Comment:
Could you make this `https://github.com/rolandriegel/nload`?   It's best to make this the projects homepage if it has one available.